Core/Object Arts/Dolphin/MVP/ViewTest.cls

```diff
@@ -10,6 +10,36 @@ ViewTest comment: ''!
 !ViewTest categoriesForClass!Unclassified! !
 !ViewTest methodsFor!
 
+allResourceIdentifiers
+
+	| badList list |
+	badList := self badResourceIdentifiers.
+	list := ResourceIdentifier allResourceIdentifiers.
+	list := list reject: [:each | badList includes: each printString].
+	^list!
+
+badResourceIdentifiers
+
+	^#(
+	)!
+
+testAllResourceIdentifiers
+	"Verify that all views actually can be opened. See #badResourceIdentifiers for failures"
+
+	true ifTrue: [^self].		"https://github.com/dolphinsmalltalk/Dolphin/pull/108"
```

Comment:
It would be useful to be able to tag tests so that they are excluded from the regression test runs wouldn't it. At the moment our only mechanism to do this is to move the tests to another package as loose method (e.g. there are packages for long running tests, known failing tests). However these are black holes in which things get lost. A simpler approach would be just to use method categories. Even better an SUnit TestSuite should be constructible from a BrowserEnvironment. I'll raise an issue for this.